src/main/java/com/airlines/entity/Airplane.java

```diff
@@ -0,0 +1,87 @@
+package com.airlines.entity;
+
+import java.time.LocalDate;
+
+public class Airplane {
+    private final int id;
+    private final String codeName;
+    private final String model;
+    private final LocalDate manufacture;
```

Comment:
this variable represent some date - so it should be 
manufactureDate

Reply:
fixed in 8d230f9c3c3300d7da8f899a28eb4137f3bf8074